data/eks_dockerimage-replication/versions/1.21.yaml

```diff
@@ -0,0 +1,10 @@
+ami:
```

Comment:
Is there a need to support 1.21 going foward?

Reply:
There is no support from eks for any versions under 1.23 ATM. So deleting them